cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,36 @@
 describe('', () => {
-  before(() => {
-
+  let user;
+  let article;
+  beforeEach(() => {
+    cy.task('generateUser').then(generateUser => {
+    user = generateUser;
+})
+    cy.login(user.email, user.username, user.password);
+    cy.task('generateArticle').then(generateArticle => {
+    article = generateArticle;
+})
   });
 
-  it('', () => {
-
+  it('should provide an ability to create new article', () => {
+    cy.login(user.email, user.username, user.password);
```

Comment:
![image](https://github.com/mate-academy/qa_cypress_e2e_article_flow/assets/85580286/8772c3f6-a272-4fa4-b280-6543c9f96c78)
this error always appears